samples/XAMLCatalog/XAMLCatalog/UIKitControls/UILabelViewController.h

```diff
@@ -17,5 +17,30 @@
 
 #import "MenuTableViewController.h"
 
-@interface UILabelViewController : MenuTableViewController
+@interface UILabelViewController : UIViewController <UITextFieldDelegate>
```

Comment:
I also realized generic viewcontrollers might not be as adaptive as tables on various form factors. It's one reason I think we used it in WOCCatalog etc.

---
In reply to: [104795774](https://github.com/Microsoft/WinObjC/pull/2178#discussion_r104795774) [](ancestors = 104795774)